app/src/main/java/org/openobservatory/ooniprobe/model/database/Measurement.java

```diff
@@ -114,10 +116,26 @@ public static File getEntryFile(Context c, int measurementId, String test_name)
 		return new File(getMeasurementDir(c), measurementId + "_" + test_name + ".json");
 	}
 
+	public void deleteEntryFile(Context c){
+		try {
+			Measurement.getEntryFile(c, this.id, this.test_name).delete();
+		} catch (Exception e) {
+			e.printStackTrace();
```

Comment:
What is the rationale of suppressing the error? Do we want to tell the user we cannot delete a file? Do we change our accounting of used disk space after this function returns? Do we care that we could not delete a file and hence the disk space we think we're using is less than the one we're really using?

Reply:
The file deletion is silent in both apps, you delete db object and files, no error messages have been told me to be included.
If you think they should please open an issue.

Reply:
If this function fails means that the file doesn't exists, so even more we shouldn't care and be silent.